store/type-map.js

```diff
@@ -994,6 +1001,45 @@ export const getters = {
     };
   },
 
+  haveComponent(state, getters) {
```

Comment:
This should be named `hasComponent`, to link the fn above

Reply:
Done